_includes/docs/user-guide/ui/mail-settings.md

```diff
@@ -174,7 +174,7 @@ To check if everything works click "Send test email" button. Keep in mind that e
 
 ### Office 365 configuration with OAuth2 authentication example
 
-Starting from Thingsboard 3.5.2 it is possible to use OAuth2 authorization for Office 365 SMTP server. 
+Starting from ThingsBoard 3.6 it is possible to use OAuth2 authorization for Office 365 SMTP server. 
```

Comment:
Why the version is changed to 3.6?

Reply:
Version 3.5.2 was not released. Following version 3.5.1 is 3.6.
In the release notes for 3.6, it is mentioned that this feature has been added.